bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -484,7 +486,7 @@ attributes from the config_cache.xml file (with keys converted to upper-case).
 <!-- The default filenames are given relative to the root directory
      for the CLM2 data in the CESM distribution -->
 <!-- Plant function types (relative to {csmdata}) -->
-<paramfile phys="clm5_1" >lnd/clm2/paramdata/ctsm51_params.c240105.nc</paramfile>
+<paramfile phys="clm5_1" >/glade/campaign/cgd/tss/people/oleson/modify_param/ctsm51_params.c240201.nc</paramfile>
```

Comment:
Right, that will need to be done by someone with permissions (I don't have them).

Reply:
I'm happy to do it. Please remind me when you're ready.

Reply:
I've now created new parameters files for clm51, clm50, and clm45 that have snow5d_thresh_for_onset on them:
ctsm51_params.c240208.nc
clm50_params.c240208.nc
clm45_params.c240208.nc
ctsm51_ciso_cwd_hr_params.c240208.nc

clm50 and clm45 aux_clm tests are bfb because snow5d_thresh_for_onset is set to its original value.
NOTE that I started from the parameter files that @samsrabin generated yesterday (PR #2350 ).

Reply:
I redid the parameter files so that they are based on @samsrabin 's latest files (c240207b) and repeated the testing.  Same (good) results as reported previously.